Modules/Findg2tmpl.cmake

```diff
@@ -26,3 +26,6 @@ if(DEFINED ENV{G2TMPL_LIBd} )
     endif()
   endforeach()
 endif()
+
+find_package_handle_standard_args(g2tmpl
```

Comment:
@kgerheiser I think we need to update this PR - shouldn't look for g2tmpl_path_d but g2tmpl_path, correct? I'll merge the rename g2tmpl PRs first and wait for this one to be fixed.

Reply:
Yes